popSimManu.tex

```diff
@@ -145,15 +145,24 @@ \section*{Introduction}
 benchmark the quality of inferences gleaned from computational population genetics.
 As large databases of population genetic variation begin to be used to inform public health procedures,
 the accuracy and quality of these inferences is becoming ever more important.
+%ALG: cite large databases of population genetic variate to inform public health procedures? UK biobank, 23andMe???
 %CK: this first paragraph is super long - might be helpful to break up
+%ALG: I think breaking up here is fine.
 
 Assessing accuracy in population genetics is challenging in large part because
  ``ground-truth'' in population genetics comes from simulation,
 not from empirical observations, and as a field there has been no agreement
-on community standards or best practices. The general modus operandi to date has been to
-introduce methods that are validated by a set of bespoke simulations produced by individual
-groups. Beyond inefficiency from duplicated effort, such a situation leads
+on community standards or best practices. 
+%ALG: Can we add an explination of why simulations are used as ground-truth?
```

Comment:
what if we added a phrase like "...not from empirical observations, as historical processes cannot be observed directly, and as a..."?

Reply:
Currently says
```
Assessing the accuracy of inference methods for population genetics is
challenging in large part because the ``ground-truth'' in question
generally comes not from direct empirical observations, as the relevant
historical processes can rarely be observed, but instead from simulations.
Population genetic simulations are therefore critically important to the
field, yet there has been no systematic attempt to establish community
standards or best practices for executing them. Instead, the general modus
```

Reply:
I think that conveys what I wanted better.